images/linux/ubuntuminimal.pkr.hcl

```diff
@@ -258,7 +258,8 @@ build {
     scripts          = [
                         "${path.root}/scripts/installers/git.sh",
                         "${path.root}/scripts/installers/github-cli.sh",
-                        "${path.root}/scripts/installers/zstd.sh"
+                        "${path.root}/scripts/installers/zstd.sh",
```

Comment:
I do not think ubuntu minimal image need runner agent installed, it is for those who build their own images